docs/pages/dsHost/ds-host_L3_Low-Level_TestSpec.md

```diff
@@ -0,0 +1,91 @@
+Below are top test use-case for the video port.
+
+|#|Test-case|Description|HAL APIs|Source|Sink|
+|-|---------|-----------|--------|------|----|
+|1|Verify that the get temperature function |After the device is running for sometime, call getTemperature, and call it after a five minute wait. The two temperatures should not see any noticable changes in temperature |`dsGetCPUTemperature()`|`Y`|`Y`|
+
+## Level 3 Python Test Cases High Level Overview
+
+The class diagram below illustrates the flow of dsHost L3 Python test cases:
+
+```mermaid
+---
+title: dsHost - Python Class Flow
+---
+classDiagram
+    testControl <|-- ut_raft : inherits
+    class ut_raft{
+    }
+    ut_raft <|-- L3_TestClasses : inherits
+    L3_TestClasses ..> dsHost : uses
+    note for testControl "uses rackConfig.yaml and deviceConfig.yaml"
+    note for dsHost "uses platformProfile.yaml"
+    note for L3_TestClasses "uses testSetupConfig.yaml"
+    note for ut_raft "suite Navigator uses testSuite.yaml"
+```
+
+- **testControl**
+  - Test Control Module for running rack Testing. This module configures the `DUT` based on the rack configuration file provided to the test.
+  - This class is defined in `RAFT` framework. For more details refer [RAFT](https://github.com/rdkcentral/python_raft/blob/1.0.0/README.md)
+- **ut_raft**
+  - Python based testing framework for writing engineering tests.
+  - It provides common functionalities like menu navigation, configuration reader, reading user response etc.
+  - For more details [ut-raft](https://github.com/rdkcentral/ut-raft).
+- **dsHost**
+  - This is test helper class which communicates with the `L3` C/C++ test running on the `DUT` through menu
+- **L3_TestClasses**
+  - These are the L3 test case classes
+  - Each class covers the each test use-case defined in [L3 Test use-cases](#level-3-test-cases-high-level-overview) table
+
+### YAML File Inputs
+
+- **rackConfig.yaml**
+  - Identifies the rack configuration and platform used
+  - References platform-specific config from `deviceConfig.yaml`
+  - For more details refer [RAFT](https://github.com/rdkcentral/python_raft/blob/1.0.0/README.md) and [example_rack_config.yml](https://github.com/rdkcentral/python_raft/blob/1.0.0/examples/configs/example_rack_config.yml)
+
+- **deviceConfig.yaml**
+  - Specifies overall configuration for the platform
+  - Can be overridden by:
+    - Changing locally .yaml file directory
+    - Using --deviceConfig command line switch
+  - For more details refer [RAFT](https://github.com/rdkcentral/python_raft/blob/1.0.0/README.md) and [example_device_config.yml](https://github.com/rdkcentral/python_raft/blob/1.0.0/examples/configs/example_device_config.yml)
+
+- **testSetupConfig.yaml**
+  - This configuration file contains the list of requirements for tests to execute. Eg: Copying the streams, setting environment variables etc.
+  - Example configuration file [dsHost_L3_testSetup.yml](../../../host/tests/dsHost_L3_Tests/dsHost_L3_testSetup.yml)
+
```

Comment:
componentProfile.yaml/platformProfile.yaml
section is missing

Reply:
Should be updated.